sdks/python/apache_beam/ml/inference/base_test.py

```diff
@@ -133,14 +133,14 @@ def test_timing_metrics(self):
             MetricsFilter().with_name('inference_batch_latency_micro_secs')))
```

Comment:
I think this metric should be renamed if we're changing the precision

Reply:
No this stays the same. I think you are mistaking the inference latency -- this one in microseconds, with the model loading latency, which is currently just called load_model_latency.

Reply:
Yeah, that makes sense.

Because I changed the fake clock to return seconds instead of microseconds, my asserts changed. Sorry for the confusion.

Also I changed this to keep the number simple.